Tue Jun 24 12:36:15 CST 2025 /Users/jrc/develop/timelink-kleio/tests
Comparing translation results.
Tue Jun 24 12:43:18 CST 2025 /Users/jrc/develop/timelink-kleio/tests
Only in kleio-home/sources/reference_translations: .gitignore
diff -r kleio-home/sources/reference_translations/bugs/bugs.err kleio-home/sources/test_translations/bugs/bugs.err
diff -r kleio-home/sources/reference_translations/bugs/bugs.files.json kleio-home/sources/test_translations/bugs/bugs.files.json
diff -r kleio-home/sources/reference_translations/bugs/bugs.rpt kleio-home/sources/test_translations/bugs/bugs.rpt
diff -r kleio-home/sources/reference_translations/bugs/bugs.xml kleio-home/sources/test_translations/bugs/bugs.xml
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.err kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.err
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.files.json kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.files.json
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.rpt kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.rpt
diff -r kleio-home/sources/reference_translations/identifications/mhk_identification_toliveira.xml kleio-home/sources/test_translations/identifications/mhk_identification_toliveira.xml
diff -r kleio-home/sources/reference_translations/issues/issue1/issue1.files.json kleio-home/sources/test_translations/issues/issue1/issue1.files.json
diff -r kleio-home/sources/reference_translations/issues/issue1/issue1.rpt kleio-home/sources/test_translations/issues/issue1/issue1.rpt
diff -r kleio-home/sources/reference_translations/issues/issue1/issue1.xml kleio-home/sources/test_translations/issues/issue1/issue1.xml
diff -r kleio-home/sources/reference_translations/issues/issue10/issue10.cli kleio-home/sources/test_translations/issues/issue10/issue10.cli
<    fonte$visita1690/1690/devassa/AUC III-d,1,5,4,24/loc=AUC
> kleio$
>     fonte$visita1690/1690/devassa/ref=AUC III-d,1,5,4,24/loc=AUC
<       devassa$dev1690/7/9/1690/fol=?/visdor=luis alves pinto/obs=luis alves pinto%o visitador assina a sua propria residencia#o visitador e o paroco sao a mesma pessoa
>         devassa$dev1690/7/9/1690/fol=?/visdor=luis alves pinto/obs=luis alves pinto%o visitador assina a sua propria residencia#o visitador e o paroco sao a mesma pessoa
<          testo$manuel fernandes/id=dev1690-per1
<             ls$profissao/serralheiro
<             ls$residencia/soure
<             ls$idade/60
>             testo$manuel fernandes/id=dev1690-per1
>                 ls$profissao/serralheiro
>                 ls$residencia/soure
>                 ls$idade/60
<             acus$1690-c1-amanc/id=dev1690-per1-obj1
>                 acus$1690-c1-amanc/id=dev1690-per1-obj1
<                caso$1690-c1-amanc/amanc/obs=ela vive longe do marido,numa casa por conta dele. ja acusados em visitas anteriores,relacao com muitos anos
>                 caso$1690-c1-amanc/amanc/obs=ela vive longe do marido,numa casa por conta dele. ja acusados em visitas anteriores,relacao com muitos anos
<                   acusado$luis da rocha manrique/id=1690-c1-amanc-per2
<                      ls$ec/c
<                      ls$residencia/soure
>                 acusado$luis da rocha manrique/id=1690-c1-amanc-per2
>                     ls$ec/c
>                     ls$residencia/soure
<                   acusada$maria simoes/id=1690-c1-acus1
<                      ls$ec/c
<                      ls$residencia/soure
>                 acusada$maria simoes/id=1690-c1-acus1
>                     ls$ec/c
>                     ls$residencia/soure
<          testo$joao mendes/id=dev1690-per3
<             ls$profissao/pedreiro
<             ls$residencia/soure
<             ls$idade/30
<             rel$parentesco/cunhado/maria simoes/1690-c1-acus1
>             testo$joao mendes/id=dev1690-per3
>                 ls$profissao/pedreiro
>                 ls$residencia/soure
>                 ls$idade/30
>                 rel$parentesco/cunhado/maria simoes/1690-c1-acus1
<             acus$1690-c1-amanc/id=dev1690-per3-obj2
>                 acus$1690-c1-amanc/id=dev1690-per3-obj2
<          testo$joao da fonseca/id=dev1690-per4
<             ls$profissao/ferrador
<             ls$residencia/soure
<             ls$idade/40
>             testo$joao da fonseca/id=dev1690-per4
>                 ls$profissao/ferrador
>                 ls$residencia/soure
>                 ls$idade/40
<             acus$1690-c1-amanc/id=dev1690-per4-obj3
>                 acus$1690-c1-amanc/id=dev1690-per4-obj3
<          testo$manuel carvalho/id=dev1690-per5
<             ls$profissao/serralheiro
<             ls$residencia/soure
<             ls$idade/32
>             testo$manuel carvalho/id=dev1690-per5
>                 ls$profissao/serralheiro
>                 ls$residencia/soure
>                 ls$idade/32
<             acus$1690-c1-amanc/id=dev1690-per5-obj4
>                 acus$1690-c1-amanc/id=dev1690-per5-obj4
<          testo$manuel rodrigues/id=dev1690-per6
<             ls$profissao/serralheiro
<             ls$residencia/soure
<             ls$idade/32
>             testo$manuel rodrigues/id=dev1690-per6
>                 ls$profissao/serralheiro
>                 ls$residencia/soure
>                 ls$idade/32
<             acus$1690-c1-amanc/id=dev1690-per6-obj5
>                 acus$1690-c1-amanc/id=dev1690-per6-obj5
<          testa$isabel pedroso/id=1009
<             ls$ec/c
<             ls$idade/35
>             testa$isabel pedroso/id=1009
>                 ls$ec/c
>                 ls$idade/35
<             acus$1690-c1-amanc/id=1009-obj6
>                 acus$1690-c1-amanc/id=1009-obj6
<             referido$antonio rodrigues/id=1009-per7
<                ls$ec/c
<                rel$parentesco/marido/isabel pedroso/1008
>                 referido$antonio rodrigues/id=1009-per7
>                 ls$ec/c
>                 rel$parentesco/marido/isabel pedroso/1008
<          testo$manuel lourenco/id=dev1690-per8
>             testo$manuel lourenco/id=dev1690-per8
<          testo$domingos simoes/id=dev1690-per9
>             testo$domingos simoes/id=dev1690-per9
<          testo$manuel de oliveira/id=dev1690-per10
>             testo$manuel de oliveira/id=dev1690-per10
<          testo$jorge goncalves/id=dev1690-per11
<             ls$residencia/pinheiro
>             testo$jorge goncalves/id=dev1690-per11
>                 ls$residencia/pinheiro
<          testo$antonio dinis/id=dev1690-per12
<             ls$residencia/pinheiro
>             testo$antonio dinis/id=dev1690-per12
>                 ls$residencia/pinheiro
<          testa$catarina gomes/id=dev1690-per13
>             testa$catarina gomes/id=dev1690-per13
<          testo$domingos marques/id=dev1690-per14
>             testo$domingos marques/id=dev1690-per14
<          testo$joao luis/id=dev1690-per15
<             ls$profissao/carpinteiro
>             testo$joao luis/id=dev1690-per15
>                 ls$profissao/carpinteiro
<          testa$francisca de andrade/id=dev1690-per16
>             testa$francisca de andrade/id=dev1690-per16
<          testo$joao francisco/id=dev1690-per17
<             ls$profissao/carpinteiro
>             testo$joao francisco/id=dev1690-per17
>                 ls$profissao/carpinteiro
<          testo$joao pimentel/id=dev1690-per18
>             testo$joao pimentel/id=dev1690-per18
<          testo$joao rodrigues/id=dev1690-per19
>             testo$joao rodrigues/id=dev1690-per19
<          testo$jorge soeiro/id=dev1690-per20
>             testo$jorge soeiro/id=dev1690-per20
<          testo$isabel nunes/id=dev1690-per21
>             testo$isabel nunes/id=dev1690-per21
<          testo$luis da silveira/id=dev1690-per22
>             testo$luis da silveira/id=dev1690-per22
<          testo$joao rodrigues/id=dev1690-per23
<             ls$profissao/barbeiro
>             testo$joao rodrigues/id=dev1690-per23
>                 ls$profissao/barbeiro
<          testo$pascoal rodrigues/id=dev1690-per24
>             testo$pascoal rodrigues/id=dev1690-per24
<          testa$brites soeiro/id=dev1690-per25
>             testa$brites soeiro/id=dev1690-per25
<          testo$matias francisco/id=dev1690-per26
>             testo$matias francisco/id=dev1690-per26
<          testo$manuel rodrigues/id=dev1690-per27
<             ls$profissao/sapateiro
>             testo$manuel rodrigues/id=dev1690-per27
>                 ls$profissao/sapateiro
<          testo$francisco dias/id=dev1690-per28
>             testo$francisco dias/id=dev1690-per28
<          testo$belchior rodrigues/id=dev1690-per29
>             testo$belchior rodrigues/id=dev1690-per29
<          testo$antonio goncalves/id=dev1690-per30
>             testo$antonio goncalves/id=dev1690-per30
diff -r kleio-home/sources/reference_translations/issues/issue10/issue10.err kleio-home/sources/test_translations/issues/issue10/issue10.err
< 0  errors. 
> 2  errors. 
diff -r kleio-home/sources/reference_translations/issues/issue10/issue10.files.json kleio-home/sources/test_translations/issues/issue10/issue10.files.json
<   "errors":0,
>   "errors":2,
Only in kleio-home/sources/test_translations/issues/issue10: issue10.ids
Only in kleio-home/sources/reference_translations/issues/issue10: issue10.old
diff -r kleio-home/sources/reference_translations/issues/issue10/issue10.rpt kleio-home/sources/test_translations/issues/issue10/issue10.rpt
23a24,33
> 
> ERROR: issue10.cli line 3 ** devassa unknown element: fol.
> Near lines: 3 
> Near lines: 4         devassa$dev1690/7/9/1690/fol=?/visdor=luis alves pinto/obs=luis alves pinto%o visitador assina a sua propria residencia#o visitador e o paroco sao a mesma pessoa
> 
> 
> ERROR: issue10.cli line 3 ** Error: missing element(s) in devassa(dev1690) must have: folio
> Near lines: 3 
> Near lines: 4         devassa$dev1690/7/9/1690/fol=?/visdor=luis alves pinto/obs=luis alves pinto%o visitador assina a sua propria residencia#o visitador e o paroco sao a mesma pessoa
> 
< 0  errors. 
> 2  errors. 
diff -r kleio-home/sources/reference_translations/issues/issue10/issue10.xml kleio-home/sources/test_translations/issues/issue10/issue10.xml
< <CLASS NAME="acusacoes" SUPER="object" TABLE="acusacoes" GROUP="acus">
> <CLASS NAME="acusacao" SUPER="object" TABLE="acusacoes" GROUP="acus">
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
<    <ELEMENT NAME="class" CLASS="class"><core>acusacoes</core></ELEMENT>
>    <ELEMENT NAME="class" CLASS="class"><core>acusacao</core></ELEMENT>
diff -r kleio-home/sources/reference_translations/issues/issue21.files.json kleio-home/sources/test_translations/issues/issue21.files.json
diff -r kleio-home/sources/reference_translations/issues/issue21.rpt kleio-home/sources/test_translations/issues/issue21.rpt
diff -r kleio-home/sources/reference_translations/issues/issue21.xml kleio-home/sources/test_translations/issues/issue21.xml
diff -r kleio-home/sources/reference_translations/issues/issue34.files.json kleio-home/sources/test_translations/issues/issue34.files.json
diff -r kleio-home/sources/reference_translations/issues/issue34.rpt kleio-home/sources/test_translations/issues/issue34.rpt
diff -r kleio-home/sources/reference_translations/issues/issue34.xml kleio-home/sources/test_translations/issues/issue34.xml
diff -r kleio-home/sources/reference_translations/issues/issue34b.files.json kleio-home/sources/test_translations/issues/issue34b.files.json
diff -r kleio-home/sources/reference_translations/issues/issue34b.rpt kleio-home/sources/test_translations/issues/issue34b.rpt
diff -r kleio-home/sources/reference_translations/issues/issue34b.xml kleio-home/sources/test_translations/issues/issue34b.xml
diff -r kleio-home/sources/reference_translations/issues/issue38/issue38.files.json kleio-home/sources/test_translations/issues/issue38/issue38.files.json
diff -r kleio-home/sources/reference_translations/issues/issue38/issue38.rpt kleio-home/sources/test_translations/issues/issue38/issue38.rpt
diff -r kleio-home/sources/reference_translations/issues/issue38/issue38.xml kleio-home/sources/test_translations/issues/issue38/issue38.xml
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.err kleio-home/sources/test_translations/linked_data/dehergne-a.err
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.files.json kleio-home/sources/test_translations/linked_data/dehergne-a.files.json
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.rpt kleio-home/sources/test_translations/linked_data/dehergne-a.rpt
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-a.xml kleio-home/sources/test_translations/linked_data/dehergne-a.xml
<          <core><![CDATA[Aguiar Souen, Joseph de (chin.) P. 8 Souen Jo-chô Kio-Jen To-Yin (Pf.: Suen ... Touo-In). N. 25 nov. 1714, Changshu (Tch'ang-chou), Kiang-sou. E. 21 sept. 1742]]></core>      <core><![CDATA[ était à Pékin en 1751. P. 1751 (Pf.). M. 1 janv. 1752, Pékin (Planchet p. 147). Pf. 807.]]></core>   </ELEMENT>
>          <core><![CDATA[Aguiar Souen, Joseph de (chin.) P. 8 Souen Jo-chô Kio-Jen To-Yin (Pf.: Suen ... Touo-In). N. 25 nov. 1714, Changshu (Tch'ang-chou), Kiang-sou. E. 21 sept. 1742; était à Pékin en 1751. P. 1751 (Pf.). M. 1 janv. 1752, Pékin (Planchet p. 147). Pf. 807.]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.err kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.err
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.files.json kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.files.json
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.rpt kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.rpt
diff -r kleio-home/sources/reference_translations/linked_data/dehergne-locations-1644.xml kleio-home/sources/test_translations/linked_data/dehergne-locations-1644.xml
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.err kleio-home/sources/test_translations/linked_data/linked-datanw.err
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.files.json kleio-home/sources/test_translations/linked_data/linked-datanw.files.json
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.rpt kleio-home/sources/test_translations/linked_data/linked-datanw.rpt
diff -r kleio-home/sources/reference_translations/linked_data/linked-datanw.xml kleio-home/sources/test_translations/linked_data/linked-datanw.xml
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.err kleio-home/sources/test_translations/linked_data/multiplelinks.err
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.files.json kleio-home/sources/test_translations/linked_data/multiplelinks.files.json
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.rpt kleio-home/sources/test_translations/linked_data/multiplelinks.rpt
diff -r kleio-home/sources/reference_translations/linked_data/multiplelinks.xml kleio-home/sources/test_translations/linked_data/multiplelinks.xml
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.err kleio-home/sources/test_translations/notariais/docsregiospontepisc.err
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.files.json kleio-home/sources/test_translations/notariais/docsregiospontepisc.files.json
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.rpt kleio-home/sources/test_translations/notariais/docsregiospontepisc.rpt
diff -r kleio-home/sources/reference_translations/notariais/docsregiospontepisc.xml kleio-home/sources/test_translations/notariais/docsregiospontepisc.xml
diff -r kleio-home/sources/reference_translations/notariais/notarial80.err kleio-home/sources/test_translations/notariais/notarial80.err
diff -r kleio-home/sources/reference_translations/notariais/notarial80.files.json kleio-home/sources/test_translations/notariais/notarial80.files.json
diff -r kleio-home/sources/reference_translations/notariais/notarial80.rpt kleio-home/sources/test_translations/notariais/notarial80.rpt
diff -r kleio-home/sources/reference_translations/notariais/notarial80.xml kleio-home/sources/test_translations/notariais/notarial80.xml
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.err kleio-home/sources/test_translations/notariais/notlousa1.err
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.files.json kleio-home/sources/test_translations/notariais/notlousa1.files.json
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.rpt kleio-home/sources/test_translations/notariais/notlousa1.rpt
diff -r kleio-home/sources/reference_translations/notariais/notlousa1.xml kleio-home/sources/test_translations/notariais/notlousa1.xml
diff -r kleio-home/sources/reference_translations/notariais/perdoes.err kleio-home/sources/test_translations/notariais/perdoes.err
diff -r kleio-home/sources/reference_translations/notariais/perdoes.files.json kleio-home/sources/test_translations/notariais/perdoes.files.json
diff -r kleio-home/sources/reference_translations/notariais/perdoes.rpt kleio-home/sources/test_translations/notariais/perdoes.rpt
diff -r kleio-home/sources/reference_translations/notariais/perdoes.xml kleio-home/sources/test_translations/notariais/perdoes.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bap-com-celebrantes.xml kleio-home/sources/test_translations/paroquiais/baptismos/bap-com-celebrantes.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapt1714.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapt1714.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem1.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem1.xml
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.files.json kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.rpt kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/bapteirasproblem2.xml kleio-home/sources/test_translations/paroquiais/baptismos/bapteirasproblem2.xml
<          <core><![CDATA[diz a margem casais]]></core>      <core><![CDATA[ so foram postos os oleos, visto ter sido baptizada em necessidade aos 21 de abril]]></core>   </ELEMENT>
>          <core><![CDATA[diz a margem casais; so foram postos os oleos, visto ter sido baptizada em necessidade aos 21 de abril]]></core>   </ELEMENT>
<          <core><![CDATA[vigario luis barreto de figueiredo castilho]]></core>      <core><![CDATA[ este assento so foi registado em 1781]]></core>   </ELEMENT>
>          <core><![CDATA[vigario luis barreto de figueiredo castilho; este assento so foi registado em 1781]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.files.json kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.rpt kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.xml kleio-home/sources/test_translations/paroquiais/baptismos/rogeriodomingos_bap_problem.xml
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.err kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.err
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.files.json kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.rpt kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.xml kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722-com-celebrante.xml
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.err kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.err
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.files.json kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.rpt kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/casamentos/cas1714-1722.xml kleio-home/sources/test_translations/paroquiais/casamentos/cas1714-1722.xml
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.err kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.err
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.files.json kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.rpt kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/crisma/crisma1753.xml kleio-home/sources/test_translations/paroquiais/crisma/crisma1753.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.err kleio-home/sources/test_translations/paroquiais/obitos/ob1688.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.files.json kleio-home/sources/test_translations/paroquiais/obitos/ob1688.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.rpt kleio-home/sources/test_translations/paroquiais/obitos/ob1688.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688.xml kleio-home/sources/test_translations/paroquiais/obitos/ob1688.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.err kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.files.json kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.rpt kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/ob1688b.xml kleio-home/sources/test_translations/paroquiais/obitos/ob1688b.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.err kleio-home/sources/test_translations/paroquiais/obitos/obit1714.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.files.json kleio-home/sources/test_translations/paroquiais/obitos/obit1714.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.rpt kleio-home/sources/test_translations/paroquiais/obitos/obit1714.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obit1714.xml kleio-home/sources/test_translations/paroquiais/obitos/obit1714.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.err kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.files.json kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.rpt kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obiteirasproblem.xml kleio-home/sources/test_translations/paroquiais/obitos/obiteirasproblem.xml
<          <core><![CDATA[sepultado em habito de santo antonio]]></core>   </ELEMENT>
>          <core><![CDATA[sepultado em habito de santo antonio;]]></core>   </ELEMENT>
<          <core><![CDATA[legitimada pelo pai]]></core>      <core><![CDATA[ morre em casa deste]]></core>   </ELEMENT>
>          <core><![CDATA[legitimada pelo pai; morre em casa deste]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.err kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.files.json kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.rpt kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/obitoShort_id_id.xml kleio-home/sources/test_translations/paroquiais/obitos/obitoShort_id_id.xml
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.err kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.err
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.files.json kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.files.json
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.rpt kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.rpt
diff -r kleio-home/sources/reference_translations/paroquiais/obitos/testeParentescos.xml kleio-home/sources/test_translations/paroquiais/obitos/testeParentescos.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.err kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.files.json kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.rpt kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/coja-rol-1841.xml kleio-home/sources/test_translations/roisdeconfessados/coja-rol-1841.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.err kleio-home/sources/test_translations/roisdeconfessados/rol1.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.files.json kleio-home/sources/test_translations/roisdeconfessados/rol1.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.rpt kleio-home/sources/test_translations/roisdeconfessados/rol1.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol1.xml kleio-home/sources/test_translations/roisdeconfessados/rol1.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.err kleio-home/sources/test_translations/roisdeconfessados/rol2.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.files.json kleio-home/sources/test_translations/roisdeconfessados/rol2.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.rpt kleio-home/sources/test_translations/roisdeconfessados/rol2.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol2.xml kleio-home/sources/test_translations/roisdeconfessados/rol2.xml
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.err kleio-home/sources/test_translations/roisdeconfessados/rol3.err
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.files.json kleio-home/sources/test_translations/roisdeconfessados/rol3.files.json
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.rpt kleio-home/sources/test_translations/roisdeconfessados/rol3.rpt
diff -r kleio-home/sources/reference_translations/roisdeconfessados/rol3.xml kleio-home/sources/test_translations/roisdeconfessados/rol3.xml
diff -r kleio-home/sources/reference_translations/varia/Devedores.err kleio-home/sources/test_translations/varia/Devedores.err
diff -r kleio-home/sources/reference_translations/varia/Devedores.files.json kleio-home/sources/test_translations/varia/Devedores.files.json
diff -r kleio-home/sources/reference_translations/varia/Devedores.rpt kleio-home/sources/test_translations/varia/Devedores.rpt
diff -r kleio-home/sources/reference_translations/varia/Devedores.xml kleio-home/sources/test_translations/varia/Devedores.xml
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.err kleio-home/sources/test_translations/varia/EmpFAfonso.err
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.files.json kleio-home/sources/test_translations/varia/EmpFAfonso.files.json
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.rpt kleio-home/sources/test_translations/varia/EmpFAfonso.rpt
diff -r kleio-home/sources/reference_translations/varia/EmpFAfonso.xml kleio-home/sources/test_translations/varia/EmpFAfonso.xml
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.err kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.err
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.files.json kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.files.json
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.rpt kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.rpt
diff -r kleio-home/sources/reference_translations/varia/auc-alunos-264605-A-140337-140771.xml kleio-home/sources/test_translations/varia/auc-alunos-264605-A-140337-140771.xml
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.err kleio-home/sources/test_translations/varia/auc_cartulario18.err
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.files.json kleio-home/sources/test_translations/varia/auc_cartulario18.files.json
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.rpt kleio-home/sources/test_translations/varia/auc_cartulario18.rpt
diff -r kleio-home/sources/reference_translations/varia/auc_cartulario18.xml kleio-home/sources/test_translations/varia/auc_cartulario18.xml
<          <core><![CDATA[o pao debulhado e limpo na eira]]></core>      <core><![CDATA[ o linho curtido e enxuto]]></core>      <core><![CDATA[ o vinho a bica do lagar e outras novidades no tempo do recolher]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[o pao debulhado e limpo na eira; o linho curtido e enxuto; o vinho a bica do lagar e outras novidades no tempo do recolher; primeira paga no ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[do pomar]]></core>      <core><![CDATA[ pago dentro do mosteiro com pena de dez mil reais por incumprimento]]></core>      <core><![CDATA[ vinho a bica do lagar e fruta ao pe da arvore]]></core>   </ELEMENT>
>          <core><![CDATA[do pomar; pago dentro do mosteiro com pena de dez mil reais por incumprimento; vinho a bica do lagar e fruta ao pe da arvore]]></core>   </ELEMENT>
<          <core><![CDATA[o pao debulhado e limpo na eira]]></core>      <core><![CDATA[ o linho curtido e enxuto]]></core>      <core><![CDATA[ o vinho a bica do lagar]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[o pao debulhado e limpo na eira; o linho curtido e enxuto; o vinho a bica do lagar; primeira paga no ano seguinte;]]></core>   </ELEMENT>
<          <core><![CDATA[medida nova]]></core>      <core><![CDATA[ pago no moinho por dia de s. miguel de setembro]]></core>   </ELEMENT>
>          <core><![CDATA[medida nova; pago no moinho por dia de s. miguel de setembro;]]></core>   </ELEMENT>
<          <core><![CDATA[do pao proprio do lavrador]]></core>      <core><![CDATA[ foro e racao pagos na eira]]></core>      <core><![CDATA[ primeira paga por s. miguel de setembro no ano presente, nos lugares respectivos e ao mordomo do mosteiro]]></core>   </ELEMENT>
>          <core><![CDATA[do pao proprio do lavrador; foro e racao pagos na eira; primeira paga por s. miguel de setembro no ano presente, nos lugares respectivos e ao mordomo do mosteiro]]></core>   </ELEMENT>
<          <core><![CDATA[por s. miguel de setembro]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[por s. miguel de setembro; primeira paga no ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[carneiro, cabrito e ovos pagos no mosteiro]]></core>      <core><![CDATA[ primeira paga pela pascoa do ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[carneiro, cabrito e ovos pagos no mosteiro; primeira paga pela pascoa do ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[na eira e ao tempo do seu recolher]]></core>      <core><![CDATA[ pago pelo natal]]></core>   </ELEMENT>
>          <core><![CDATA[na eira e ao tempo do seu recolher; pago pelo natal]]></core>   </ELEMENT>
<          <core><![CDATA[do pao do lavrador]]></core>      <core><![CDATA[ pao limpo e debulhado na eira]]></core>      <core><![CDATA[ o vinho a bica do lagar]]></core>      <core><![CDATA[ o linho curtido e enxuto]]></core>      <core><![CDATA[ legumes ao tempo do recolher]]></core>   </ELEMENT>
>          <core><![CDATA[do pao do lavrador; pao limpo e debulhado na eira; o vinho a bica do lagar; o linho curtido e enxuto; legumes ao tempo do recolher]]></core>   </ELEMENT>
<          <core><![CDATA[segunda e terceira vidas]]></core>      <core><![CDATA[ por s. miguel de setembro e com abatimento de pensao por benfeitorias as casas e mosteiro]]></core>   </ELEMENT>
>          <core><![CDATA[segunda e terceira vidas; por s. miguel de setembro e com abatimento de pensao por benfeitorias as casas e mosteiro]]></core>   </ELEMENT>
<          <core><![CDATA[da casa do dito mosteiro]]></core>      <core><![CDATA[ assinou pelo foreiro]]></core>   </ELEMENT>
>          <core><![CDATA[da casa do dito mosteiro; assinou pelo foreiro]]></core>   </ELEMENT>
<          <core><![CDATA[por s. joao baptista]]></core>   </ELEMENT>
>          <core><![CDATA[por s. joao baptista;]]></core>   </ELEMENT>
<          <core><![CDATA[da moeda corrente de seis ceitis]]></core>      <core><![CDATA[ por s. miguel de setembro e no mosteiro]]></core>   </ELEMENT>
>          <core><![CDATA[da moeda corrente de seis ceitis; por s. miguel de setembro e no mosteiro]]></core>   </ELEMENT>
<          <core><![CDATA[da moeda corrente de seis ceitis ao real]]></core>      <core><![CDATA[ por s.miguel de setembro]]></core>   </ELEMENT>
>          <core><![CDATA[da moeda corrente de seis ceitis ao real; por s.miguel de setembro]]></core>   </ELEMENT>
<          <core><![CDATA[safra de dois em dois anos, feito no pote]]></core>      <core><![CDATA[ pelo entrudo]]></core>   </ELEMENT>
>          <core><![CDATA[safra de dois em dois anos, feito no pote; pelo entrudo]]></core>   </ELEMENT>
<          <core><![CDATA[a bica do lagar]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[a bica do lagar; primeira paga no ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[da moeda corrente de seis ceitis]]></core>      <core><![CDATA[ por s. miguel de setembro]]></core>   </ELEMENT>
>          <core><![CDATA[da moeda corrente de seis ceitis; por s. miguel de setembro]]></core>   </ELEMENT>
<          <core><![CDATA[da moeda corrente de seis ceitis]]></core>      <core><![CDATA[ por s. miguel de setembro]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[da moeda corrente de seis ceitis; por s. miguel de setembro; primeira paga no ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[safra de dois em dois anos]]></core>      <core><![CDATA[ primeira paga no ano seguinte]]></core>   </ELEMENT>
>          <core><![CDATA[safra de dois em dois anos; primeira paga no ano seguinte]]></core>   </ELEMENT>
<          <core><![CDATA[pelo aluguer do carreto por cada quarteirao da racao]]></core>      <core><![CDATA[ pagam lopo anes e sua mulher branca anes]]></core>   </ELEMENT>
>          <core><![CDATA[pelo aluguer do carreto por cada quarteirao da racao; pagam lopo anes e sua mulher branca anes]]></core>   </ELEMENT>
<          <core><![CDATA[por cada quarteiro de racao de carreto]]></core>      <core><![CDATA[ pagam francisco anes e sua filha catarina lopes]]></core>   </ELEMENT>
>          <core><![CDATA[por cada quarteiro de racao de carreto; pagam francisco anes e sua filha catarina lopes]]></core>   </ELEMENT>
<          <core><![CDATA[pagam francisco anes e sua filha catarina lopes]]></core>      <core><![CDATA[ pao debulhado, limpo e pago na eira]]></core>      <core><![CDATA[ aves e ovos pagos no mosteiro por s. miguel de setembro]]></core>      <core><![CDATA[ o vinho a bica do lagar]]></core>      <core><![CDATA[ o linho curido e enxuto]]></core>   </ELEMENT>
>          <core><![CDATA[pagam francisco anes e sua filha catarina lopes; pao debulhado, limpo e pago na eira; aves e ovos pagos no mosteiro por s. miguel de setembro; o vinho a bica do lagar; o linho curido e enxuto]]></core>   </ELEMENT>
<          <core><![CDATA[por s. miguel de setembro]]></core>      <core><![CDATA[ seis mil reais por incumprimento]]></core>   </ELEMENT>
>          <core><![CDATA[por s. miguel de setembro; seis mil reais por incumprimento]]></core>   </ELEMENT>
<          <core><![CDATA[aforamento para sempre celerado entre mosteiro de santa clara e andre afonso, de meio casal no qual mora]]></core>      <core><![CDATA[ um quarto de casal]]></core>      <core><![CDATA[ um oitavo de casal]]></core>      <core><![CDATA[ dois pedacos de mato, sitos em orelhudo]]></core>   </ELEMENT>
>          <core><![CDATA[aforamento para sempre celerado entre mosteiro de santa clara e andre afonso, de meio casal no qual mora; um quarto de casal; um oitavo de casal; dois pedacos de mato, sitos em orelhudo]]></core>   </ELEMENT>
<          <core><![CDATA[pao limpo na eira]]></core>      <core><![CDATA[ legumes]]></core>      <core><![CDATA[ vinho a bica do lagar]]></core>      <core><![CDATA[ linho curtido e enxuto no tendal]]></core>   </ELEMENT>
>          <core><![CDATA[pao limpo na eira; legumes; vinho a bica do lagar; linho curtido e enxuto no tendal]]></core>   </ELEMENT>
<          <core><![CDATA[emprazamento em tres vidas celebrado entre mosteiro de santa clara e alvaro botelho caldeira, de uns moinhos de moer pao]]></core>      <core><![CDATA[ um engenho que foi lagar de fazer azeite]]></core>      <core><![CDATA[ um pomar e umas casas com seu lagar para fazer vinho]]></core>      <core><![CDATA[ um pedaco de lameira]]></core>   </ELEMENT>
>          <core><![CDATA[emprazamento em tres vidas celebrado entre mosteiro de santa clara e alvaro botelho caldeira, de uns moinhos de moer pao; um engenho que foi lagar de fazer azeite; um pomar e umas casas com seu lagar para fazer vinho; um pedaco de lameira]]></core>   </ELEMENT>
<          <core><![CDATA[emprazamento em tres pessoas celebrado entre mosteiro de santa clara e jorge fernandes]]></core>      <core><![CDATA[ antonio rodrigues]]></core>      <core><![CDATA[ manuel fernandes]]></core>      <core><![CDATA[ simao rodrigues]]></core>      <core><![CDATA[ bras pires, de umas casas sitas no burgo do mosteiro de santa clara]]></core>   </ELEMENT>
>          <core><![CDATA[emprazamento em tres pessoas celebrado entre mosteiro de santa clara e jorge fernandes; antonio rodrigues; manuel fernandes; simao rodrigues; bras pires, de umas casas sitas no burgo do mosteiro de santa clara]]></core>   </ELEMENT>
<          <core><![CDATA[por s. miguel de setembro]]></core>      <core><![CDATA[ os tres primeiros inquilinos]]></core>   </ELEMENT>
>          <core><![CDATA[por s. miguel de setembro; os tres primeiros inquilinos]]></core>   </ELEMENT>
<          <core><![CDATA[ por s. miguel de setembro]]></core>      <core><![CDATA[ os dois ultimos inquilinos]]></core>   </ELEMENT>
>          <core><![CDATA[ por s. miguel de setembro; os dois ultimos inquilinos]]></core>   </ELEMENT>
<          <core><![CDATA[novo emprazamento em tres vidas celebrado entre mosteiro de santa clara e filipa leite, de umas casas, casais e outras propriedades sitas na freguesia de sta. maria do sobrado]]></core>      <core><![CDATA[ em riba douro]]></core>      <core><![CDATA[ na freguesia de sampaio, termo da vila de guimaraes]]></core>      <core><![CDATA[ na freguesia de s. martinho de butarim e dos pi�e]]></core>      <core><![CDATA[ na freguesia de santa maria de quintas, no termo de barcelos]]></core>   </ELEMENT>
>          <core><![CDATA[novo emprazamento em tres vidas celebrado entre mosteiro de santa clara e filipa leite, de umas casas, casais e outras propriedades sitas na freguesia de sta. maria do sobrado; em riba douro; na freguesia de sampaio, termo da vila de guimaraes; na freguesia de s. martinho de butarim e dos pi�e; na freguesia de santa maria de quintas, no termo de barcelos]]></core>   </ELEMENT>
<          <core><![CDATA[novo aforamento em fatiosim celebrado entre mosteiro de santa clara e andre fernandes]]></core>      <core><![CDATA[ andre bras e simao pires, de uns matos com agua para moendas]]></core>   </ELEMENT>
>          <core><![CDATA[novo aforamento em fatiosim celebrado entre mosteiro de santa clara e andre fernandes; andre bras e simao pires, de uns matos com agua para moendas]]></core>   </ELEMENT>
<          <core><![CDATA[pao, trigo e segundas]]></core>      <core><![CDATA[ linho e legumes]]></core>   </ELEMENT>
>          <core><![CDATA[pao, trigo e segundas; linho e legumes]]></core>   </ELEMENT>
<          <core><![CDATA[aforamento em fatiosim para sempre celebrado entre mosteiro de santa clara e goncalo afonso]]></core>      <core><![CDATA[ luis afonso]]></core>      <core><![CDATA[ joao afonso]]></core>      <core><![CDATA[ domingos afonso]]></core>      <core><![CDATA[ rodrigo alvares]]></core>      <core><![CDATA[ simao rodrigues]]></core>      <core><![CDATA[ fernao gomes]]></core>      <core><![CDATA[ simao vaz]]></core>      <core><![CDATA[ joao afonso]]></core>      <core><![CDATA[ domingos vaz]]></core>      <core><![CDATA[ francisco pires]]></core>      <core><![CDATA[ francisco fernandes]]></core>      <core><![CDATA[ catarina vaz]]></core>      <core><![CDATA[ joao rodrigues, de umas terras, vinhas, currais, herancas e matos, sitos em gatoes, montemor-o-velho]]></core>   </ELEMENT>
>          <core><![CDATA[aforamento em fatiosim para sempre celebrado entre mosteiro de santa clara e goncalo afonso; luis afonso; joao afonso; domingos afonso; rodrigo alvares; simao rodrigues; fernao gomes; simao vaz; joao afonso; domingos vaz; francisco pires; francisco fernandes; catarina vaz; joao rodrigues, de umas terras, vinhas, currais, herancas e matos, sitos em gatoes, montemor-o-velho]]></core>   </ELEMENT>
diff -r kleio-home/sources/reference_translations/varia/cartas.err kleio-home/sources/test_translations/varia/cartas.err
diff -r kleio-home/sources/reference_translations/varia/cartas.files.json kleio-home/sources/test_translations/varia/cartas.files.json
diff -r kleio-home/sources/reference_translations/varia/cartas.rpt kleio-home/sources/test_translations/varia/cartas.rpt
diff -r kleio-home/sources/reference_translations/varia/cartas.xml kleio-home/sources/test_translations/varia/cartas.xml
diff -r kleio-home/sources/reference_translations/varia/dehergne-residences.err kleio-home/sources/test_translations/varia/dehergne-residences.err
diff -r kleio-home/sources/reference_translations/varia/dehergne-residences.files.json kleio-home/sources/test_translations/varia/dehergne-residences.files.json
diff -r kleio-home/sources/reference_translations/varia/dehergne-residences.rpt kleio-home/sources/test_translations/varia/dehergne-residences.rpt
diff -r kleio-home/sources/reference_translations/varia/dehergne-residences.xml kleio-home/sources/test_translations/varia/dehergne-residences.xml
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.err kleio-home/sources/test_translations/varia/eleicoes1.err
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.files.json kleio-home/sources/test_translations/varia/eleicoes1.files.json
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.rpt kleio-home/sources/test_translations/varia/eleicoes1.rpt
diff -r kleio-home/sources/reference_translations/varia/eleicoes1.xml kleio-home/sources/test_translations/varia/eleicoes1.xml
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.err kleio-home/sources/test_translations/varia/genealdiversas.err
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.files.json kleio-home/sources/test_translations/varia/genealdiversas.files.json
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.rpt kleio-home/sources/test_translations/varia/genealdiversas.rpt
diff -r kleio-home/sources/reference_translations/varia/genealdiversas.xml kleio-home/sources/test_translations/varia/genealdiversas.xml
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.err kleio-home/sources/test_translations/varia/glopes-lap1561.err
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.files.json kleio-home/sources/test_translations/varia/glopes-lap1561.files.json
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.rpt kleio-home/sources/test_translations/varia/glopes-lap1561.rpt
diff -r kleio-home/sources/reference_translations/varia/glopes-lap1561.xml kleio-home/sources/test_translations/varia/glopes-lap1561.xml
diff -r kleio-home/sources/reference_translations/varia/ivcc.err kleio-home/sources/test_translations/varia/ivcc.err
diff -r kleio-home/sources/reference_translations/varia/ivcc.files.json kleio-home/sources/test_translations/varia/ivcc.files.json
diff -r kleio-home/sources/reference_translations/varia/ivcc.rpt kleio-home/sources/test_translations/varia/ivcc.rpt
diff -r kleio-home/sources/reference_translations/varia/ivcc.xml kleio-home/sources/test_translations/varia/ivcc.xml
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.err kleio-home/sources/test_translations/varia/lrazao516pe.err
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.files.json kleio-home/sources/test_translations/varia/lrazao516pe.files.json
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.rpt kleio-home/sources/test_translations/varia/lrazao516pe.rpt
diff -r kleio-home/sources/reference_translations/varia/lrazao516pe.xml kleio-home/sources/test_translations/varia/lrazao516pe.xml
diff -r kleio-home/sources/reference_translations/varia/misMesa.err kleio-home/sources/test_translations/varia/misMesa.err
diff -r kleio-home/sources/reference_translations/varia/misMesa.files.json kleio-home/sources/test_translations/varia/misMesa.files.json
diff -r kleio-home/sources/reference_translations/varia/misMesa.rpt kleio-home/sources/test_translations/varia/misMesa.rpt
diff -r kleio-home/sources/reference_translations/varia/misMesa.xml kleio-home/sources/test_translations/varia/misMesa.xml
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.err kleio-home/sources/test_translations/varia/mp758-santiago-beduido.err
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.files.json kleio-home/sources/test_translations/varia/mp758-santiago-beduido.files.json
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.rpt kleio-home/sources/test_translations/varia/mp758-santiago-beduido.rpt
diff -r kleio-home/sources/reference_translations/varia/mp758-santiago-beduido.xml kleio-home/sources/test_translations/varia/mp758-santiago-beduido.xml
diff -r kleio-home/sources/reference_translations/varia/nommiz.err kleio-home/sources/test_translations/varia/nommiz.err
diff -r kleio-home/sources/reference_translations/varia/nommiz.files.json kleio-home/sources/test_translations/varia/nommiz.files.json
diff -r kleio-home/sources/reference_translations/varia/nommiz.rpt kleio-home/sources/test_translations/varia/nommiz.rpt
diff -r kleio-home/sources/reference_translations/varia/nommiz.xml kleio-home/sources/test_translations/varia/nommiz.xml
diff -r kleio-home/sources/reference_translations/varia/quotes.err kleio-home/sources/test_translations/varia/quotes.err
diff -r kleio-home/sources/reference_translations/varia/quotes.files.json kleio-home/sources/test_translations/varia/quotes.files.json
diff -r kleio-home/sources/reference_translations/varia/quotes.rpt kleio-home/sources/test_translations/varia/quotes.rpt
diff -r kleio-home/sources/reference_translations/varia/quotes.xml kleio-home/sources/test_translations/varia/quotes.xml
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.err kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.err
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.files.json kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.files.json
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.rpt kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.rpt
diff -r kleio-home/sources/reference_translations/varia/sourecoimbrabispcoimbra_apontamentosd.xml kleio-home/sources/test_translations/varia/sourecoimbrabispcoimbra_apontamentosd.xml
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.err kleio-home/sources/test_translations/varia/test-atr-date.err
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.files.json kleio-home/sources/test_translations/varia/test-atr-date.files.json
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.rpt kleio-home/sources/test_translations/varia/test-atr-date.rpt
diff -r kleio-home/sources/reference_translations/varia/test-atr-date.xml kleio-home/sources/test_translations/varia/test-atr-date.xml
diff -r kleio-home/sources/reference_translations/varia/vereacao.err kleio-home/sources/test_translations/varia/vereacao.err
diff -r kleio-home/sources/reference_translations/varia/vereacao.files.json kleio-home/sources/test_translations/varia/vereacao.files.json
diff -r kleio-home/sources/reference_translations/varia/vereacao.rpt kleio-home/sources/test_translations/varia/vereacao.rpt
diff -r kleio-home/sources/reference_translations/varia/vereacao.xml kleio-home/sources/test_translations/varia/vereacao.xml
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.err kleio-home/sources/test_translations/varia/xpto_comuns_soure.err
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.files.json kleio-home/sources/test_translations/varia/xpto_comuns_soure.files.json
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.rpt kleio-home/sources/test_translations/varia/xpto_comuns_soure.rpt
diff -r kleio-home/sources/reference_translations/varia/xpto_comuns_soure.xml kleio-home/sources/test_translations/varia/xpto_comuns_soure.xml
Comparing translation results finished.
